ONSdigital/respondent-home-ui
# Use casesvc sampleunitid endpoint

# Motivation and Context
The casesvc endpoint that was being used to get the list of cases did not return the expected result when run in preprod. A different endpoint is now being used to get the cases in a single request, rather than a request for each case.

# How to test?
Have EQ running and create a collection exercise. Ensure you start some cases so they are marked as INPROGRESS then run the script. You should see a 200 response in the log for all the cases you have started.

# Links
https://trello.com/c/lNmyBmPH
